webapp/store.md

```diff
@@ -1,7 +0,0 @@
-# STORE
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-11-04T14:00:27Z" title="Monday, November 4th 2019, 3:00:27 pm +01:00">Nov 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,7 +0,0 @@
-# STORE
```


Why is this README file removed?